lib/wipe_out/plans/dsl.rb

```diff
@@ -72,6 +72,7 @@ def relation(name, plan = nil, plans: nil, &block)
           plan = plan.plan if plan.is_a?(BuiltPlan)
           dsl = Dsl.new(plan)
           dsl.instance_exec(&block) if block.present?
+          @plan.callbacks.each { |callback| plan.add_callback(callback) }
```

Comment:
I can see it's in the if/else branch with condition on `plans` presence. Are relations with `plans` specified safe to use then?